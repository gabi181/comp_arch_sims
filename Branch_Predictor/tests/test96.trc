2 3 6 2 global_history global_tables not_using_share
0x29074C50 T 0x550A7288
0x29074C50 T 0x550A7288
0x29074C50 N 0x550A7288
0x29074C50 N 0x550A7288
0x29074C50 N 0x550A7288
0x29074C50 N 0x550A7288
0x29074C50 N 0x550A7288
0x29074C50 N 0x550A7288
0x29074C50 N 0x550A7288
0x29074C50 N 0x550A7288
0x29074C50 N 0x550A7288
0x29074C50 N 0x550A7288
0x29074C50 T 0x550A7288
0x29074C50 T 0x550A7288
0x29074C50 N 0x550A7288
