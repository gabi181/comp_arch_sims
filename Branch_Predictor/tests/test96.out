0x29074c50 N 0x29074c54
0x29074c50 T 0x550a7288
0x29074c50 T 0x550a7288
0x29074c50 T 0x550a7288
0x29074c50 T 0x550a7288
0x29074c50 T 0x550a7288
0x29074c50 T 0x550a7288
0x29074c50 N 0x29074c54
0x29074c50 N 0x29074c54
0x29074c50 N 0x29074c54
0x29074c50 N 0x29074c54
0x29074c50 N 0x29074c54
0x29074c50 N 0x29074c54
0x29074c50 T 0x550a7288
0x29074c50 N 0x29074c54
flush_num: 7, br_num: 15, size: 93b
